webapp/src/Components/Pages/HomePage.tsx

```diff
@@ -44,65 +45,81 @@ class HomePageComponent extends React.PureComponent<Props, State> {
+        GlobalService.getInstance()
+            .getLoggedInUser()
             .then((loggedInUser) => this.props.setLoggedInUser(loggedInUser))
             .catch(() => undefined)
     }
 
     public render() {
-        const {classes, width, loggedInUser} = this.props
+        const { classes, width, loggedInUser } = this.props
 
-        const alignCenterProps: GridProps = {container: true, justify: "center", alignItems: "center"}
+        const alignCenterProps: GridProps = { container: true, justify: "center", alignItems: "center" }
         return (
             <div className={classes.backgroundContainer}>
                 <UploadDialogWrapper buttonStyle="floating">
                     <div className={classes.root}>
                         <Grid container justify="center" alignItems="flex-start" spacing={40} className={classes.child}>
-                            <Grid item xs={12} {...alignCenterProps} style={{minHeight: "300px"}} direction="column">
-                                <Logo imgStyle={{maxWidth: "80vw"}}/>
-                                {this.state.replayCount &&
-                                <>
-                                    <br/>
-                                    <Typography>
-                                        <i>{this.state.replayCount} replays and counting...</i>
-                                    </Typography>
-                                </>
-                                }
+                            <Grid item xs={12} {...alignCenterProps} style={{ minHeight: "300px" }} direction="column">
+                                <Logo imgStyle={{ maxWidth: "80vw" }} />
+                                {this.state.replayCount && (
+                                    <>
+                                        <br />
+                                        <Typography>
+                                            <i>{this.state.replayCount} replays and counting...</i>
+                                        </Typography>
+                                    </>
+                                )}
                             </Grid>
-                            <Grid item xs={11} {...alignCenterProps} style={{padding: "20px 0 20px 0"}}>
-                                <Search usePaper/>
+                            <Grid item xs={11} {...alignCenterProps} style={{ padding: "20px 0 20px 0" }}>
+                                <Search usePaper />
                             </Grid>
-                            <Grid item xs={12} sm={10} md={8} container spacing={16} alignItems="center"
-                                  style={{maxWidth: 550}}>
-                                <Grid item xs={6} style={{textAlign: "center"}}>
-                                    {loggedInUser ?
-                                        <Link to={PLAYER_PAGE_LINK(loggedInUser.id)} style={{textDecoration: "none"}}>
+                            <Grid
+                                item
```

Comment:
can we have `item xs sm md` on one line and `container spacing alignitems` on another?